.travis.yml

```diff
@@ -45,14 +45,14 @@ jobs:
       script:
         - ddev dep verify
         - ddev manifest verify --include-extras
-        - ddev test
+        - ddev test --cov
         - ddev test --bench || true
     - stage: test
       env: CHECK=datadog_checks_base PYTHON3=true
       script:
         - ddev dep verify
         - ddev manifest verify --include-extras
-        - travis_retry ddev test ${CHECK}
+        - travis_retry ddev test --cov ${CHECK}
```

Comment:
Not sure what the intention is there but note that `--cov` (if it's the option from pytest-cov) will eat up the next positional argument (you should have `--cov` as the last option). Eg: `--cov foobar` is the same as `--cov=foobar` (but `--cov --foobar` will work as expected).

Reply:
Ah, yes this is actually it's own flag. Thanks!